pandas/core/series.py

```diff
@@ -127,6 +127,39 @@ def wrapper(self):
     return wrapper
 
 
+# see gh-17261
```

Comment:
so why don't we move these to somewhere that we don't have so many dependencies on import, try 
pandas.core.construction; the key is to put the Series import inside the create_series_with_explicity_dtype.